torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20262,6 +20291,17 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
         ),
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.nn.functional.prelu",
+        torch_opinfo_name="nn.functional.prelu",
+        skips=(
+            # NumPy does not support bfloat16
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref',
```

Comment:
But it's test_numpy_ref that doesn't support bfloat16 -- I think this is just a type promotion discrepancy